cmake/common.cmake

```diff
@@ -0,0 +1,163 @@
+include(GNUInstallDirs)
+include(CMakePackageConfigHelpers)
+
+add_compile_options(-std=c++17)
+
+# Specific compile optios across all targets
+#add_compile_definitions(MINIMAL_LOGGING)
```

Comment:
@s-jain1 Is this supposed to be commented out?  Or just removed altogether?

Reply:
I have taken reference for this file from edgeai-gst-apps. And there they have commented out this line similarly.